src/completions.jl

```diff
@@ -338,6 +376,45 @@ function global_completions!(items::Dict{String, CompletionItem}, state::ServerS
     return items
 end
 
+# LaTeX and emoji completions
+# ===========================
+
+function add_emoji_latex_completions!(items::Dict{String,CompletionItem}, state::ServerState, uri::URI, pos::Position)
+    fi = get_fileinfo(state, uri)
+    fi === nothing && return items
+
+    backslash_offset = get_backslash_offset(state, uri, pos)
+    backslash_offset === nothing && return items
+    backslash_pos = offset_to_xy(fi, backslash_offset)
+
+    function create_ci(key, val, description)
+        return CompletionItem(
+            label=key,
+            labelDetails=CompletionItemLabelDetails(
+                description=description
+            ),
+            kind=CompletionItemKind.Unit,
```

Comment:
```suggestion
            kind=CompletionItemKind.Snippet,
```
`Snippet` seems to better fit here?